examples/io/plot_read_events.py

```diff
@@ -0,0 +1,77 @@
+"""
+.. _ex_read_events:
```

Comment:
But not the tutorials...? Should we at some point change all our `_tut_*` references to be the long sphinx-gallery one, too? I'd rather go the other way to use the shorter refs.